torchprime/hf_models/configs/default.yaml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  - _self_
+
+# extra env variables you want to pass to the training script
+env:
+  - NUM_SLICE: 1
+  - XLA_PERSISTENT_CACHE_PATH: "/tmp/xla_cache"
+  - PROFILE_LOGDIR: "gs://torchprime/profile"
+  - XLA_USE_SPMD: 1
+  - ENABLE_PJRT_COMPATIBILITY: true
```

Comment:
Is the `ENABLE_PJRT_COMPATIBILITY` flag needed by anything? From what I could tell, it's specific to JAX and not PyTorch/XLA.